pkg/registry/core/shoot/storage/kubeconfig.go

```diff
@@ -44,6 +44,7 @@ import (
 type KubeconfigREST struct {
 	secretLister         kubecorev1listers.SecretLister
 	internalSecretLister gardencorelisters.InternalSecretLister
+	configMapLister      kubecorev1listers.ConfigMapLister
```

Comment:
Also add a TODO statement to remove this struct attribute

Reply:
✅